File: 900-reordered-power-of-2/reordered-power-of-2.java
class Solution {
    public boolean reorderedPowerOf2(int n) {
        char[] s=sortedDigits(n);
        for(int i=0;i<30;i++){
            char[]power=sortedDigits(1<<i);
            if(Arrays.equals(s,power))return true;
        }
        return false;
    }
    public char[] sortedDigits(int n){
        char[] ans=String.valueOf(n).toCharArray();
        Arrays.sort(ans);
        return ans;
    }
}

File: 900-reordered-power-of-2/reordered-power-of-2.py
class Solution(object):
    def reorderedPowerOf2(self, n):
        """
        :type n: int
        :rtype: bool
        """
        ans=Counter(str(n))
        print(ans)
        for i in range(30):
            power=str(1<<i)
            if(ans==Counter(power)):
                return True
        return False        